### ISSUE TITLE
Proper way to get units from DB ?

### ISSUE THREAD

**Issue by fpavogt** [MeteoSwiss/dvas#140]
**Question**
@modolol: What would be the best way to access the unit field for a given profile variable ? I can get them using:
```
from .database.database import DatabaseManager

DatabaseManager().get_table(DatabaseManager().DB_TABLES[9])[0]
```

but this is evidently not ideal, because the user needs to know which index corresponds to the parameter table (i.e. 9), and what is the id of the required variable (0=time). 

I see that you have implemented a little function in the DB explorer. Would this be a better way to do this ? 

Basically just trying to see if there is something already in place that I am not aware of, or if we need a new "tool" to access that info easily.
